### ISSUE TITLE
Modification of Vanilla drops

### ISSUE THREAD

**Issue by Carolusclen** [PluginBugs/Issues-ItemsAdder#1519]
### Terms

- [X] I'm using the very latest version of ItemsAdder and its dependencies.
- [X] I already searched on this [Github page](https://github.com/PluginBugs/Issues-ItemsAdder/issues) to check if the same suggestion was already reported.
- [X] I already searched on the [plugin wiki](https://itemsadder.devs.beer/) to know if the feature is already implemented.
- [X] I already searched on the **Discord** server to know if anyone already has a solution for this or if the feature was already suggested.

### Discord tag (optional)

Inanna#4464

### Describe the solution you'd like

Aftr migrating from a previous plugin similar to this one, I have encountered a feature that I believe is no present. I have found a couple of people on the discord and 1 answered me today explaining that it may not be possible yet.

I would like the ability to disable Vanilla drops.
example:
Dirt block -> drops Dirt Block
I have created a custom item called "Pile of Dirt" and added it to "Dirt Block"
now;
Dirt Block -> drops Dirt block & Pile of dirt.

I would like an ability to stop the vanilla dirt block dropping its default dirt block and only have it drop my item.
This goes for most Vanilla blocks

### Is your feature request related to a problem?

nope

### Describe alternatives you've considered

I have seen other plugins I believe that can do this, I may need to test them and see if they work in conjunction to this plugin
I can also try using the previous plugin I migrated to this one from and just use that one to deal with the disabling of the vanilla drops.

### Additional context

N/A

**Comment by imparium-de** [comment]
We would love to see this getting implemented. Disabling default drops from vanilla blocks seems to be an oversight, when seeing that this plugin is already capable of. I also like the shown config example from https://github.com/PluginBugs/Issues-ItemsAdder/issues/2530 .

`disable_default_drop: true` seems to be a very intuitive name for such an option.

**Comment by LoneDev6** [comment]
> `disable_default_drop: true` seems to be a very intuitive name for such an option.

Yes but I don't think the loot entry is the correct place to put such option, as there might be thousand of loots for the same block and would cause confusion to have the `disable_default_drop` property possibly put on multiple loots with different value (true/false).

I think a new section, similar to the disable recipes which is in config.yml, could be added to the general yml files to let admins disable loots for particular blocks/entities.

An example:

```yml
info:
  namespace: test
overwrite_vanilla_behaviour:
  disable_loots:
    blocks:
      stone:
        enabled: true
        type: STONE
      iron_ore:
        enabled: true
        type: IRON_ORE
    mobs:
      zombie:
        enabled: true
        type: ZOMBIE
```
